Test/Machine/test_machine.py

```diff
@@ -68,6 +69,14 @@ def initializer(rng, shape):
 
     machines["Jax RbmSpinPhase (R->C)"] = nk.machine.JaxRbmSpinPhase(hi, alpha=1)
 
+    class flaxrbm(flax.linen.Module):
+        def apply(self, x):
+            x = flax.nn.Dense(x, features=2, dtype=jax.numpy.complex128)
+            x = jax.numpy.log(jax.numpy.cosh(x))
```

Comment:
let's be careful with log(cosh) instabilities in production machines

Reply:
Ah! right. I have to add [this code](https://github.com/netket/netket/blob/17a48f2cf1349398d55b676badb1d2e2babeb91c/netket/machine/jax.py#L295) to the activation functions.

Reply:
still, also this is part of the 'old' pr. I should remove that first commit.
For this PR, I still have to push (actually, mostly write) the tests...

Reply:
Addressed by defining log cosh in our own activation functions. I have to rewrite the tests anyway so will close this for now.